TA_dataset/bin/dataset_api.py

```diff
@@ -102,17 +112,28 @@ def ds_lrq_facet_values(
             filter_=filter, name=name, max_values=max_values
         ),
     )
-    return ds_lrq_run_loop(client=client, body=body)
+    if acc_conf.get("tenant") is not None:
+        tenant_value = acc_conf.get("tenant")
+        if tenant_value:
+            acc_conf = {"tenant": True}
+        else:
+            acc_conf = {"tenant": False, "accountIds": acc_conf["account_ids"]}
+    return ds_lrq_run_loop(logger, client=client, body=body)
 
 
 # Executes LRQ run loop of launch-ping-remove API requests until the query completes
 # with a result
 # Returns tuple - value, error message
 def ds_lrq_run_loop(
-    client: AuthenticatedClient, body: PostQueriesLaunchQueryRequestBody
+    log,
+    client: AuthenticatedClient,
+    body: PostQueriesLaunchQueryRequestBody,
+    acc_conf=None,
 ):
     body.query_priority = PostQueriesLaunchQueryRequestBodyQueryPriority.HIGH
-    response = post_queries.sync_detailed(client=client, json_body=body)
+    response = post_queries.sync_detailed(
+        client=client, json_body=body, tenant_details=acc_conf, logger=log
```

Comment:
Why dont we unify the naming here? `tenant_details vs acc_conf` Having 2 different name for same variable make it bit more unreadable